files/nbde_client/module-setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+depends() {
+  echo network-manager
+  return 255
+}
+
+install() {
+  inst_multiple nmcli grep cut
```

Comment:
Is this something we could add to `__nbde_client_packages`?  Or is this some sort of `dracut` thing?

Reply:
This is a `dracut` thing. We are requesting those binaries to be added to the initramfs.

Reply:
In terms of `__nbde_client_packages`, I added `NetworkManager`, to make sure `nmcli` exists.

Reply:
ok - for the `shellcheck` reported issues, just add them as suppressions at the top of the file e.g.
```
# these functions/variables are defined by dracut
# shellcheck disable=SC2154
```

Reply:
Done.